bundles/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/handler/GroheOndusHandlerFactory.java

```diff
@@ -43,21 +50,72 @@
 
     private final Map<ThingUID, @Nullable ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
 
+    private @Nullable HttpService httpService;
```

Comment:
If you have the time could you use field references instead of method references? Eg:
`@Reference private @Nullable HttpService httpService;` (and remove the methods).

Shorter code is better code. Thanks

Reply:
Sure, thanks for the hint! :)